portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,64 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+var imageIndex = 1;
+var totalimg = 7;
```

Comment:
Using var is not a good practice, use const or let.
You can setup VSCode extension to give linting suggestions
Example: https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint